cmd/component/model.go

```diff
@@ -245,6 +253,20 @@ func (m Model) Update(msg tea.Msg) (tea.Model, tea.Cmd) { //nolint:cyclop
 			case "i":
 				// TODO: [CFX-3996] What do we show here?
 				return m, m.showComponentInfo()
+			case tea.KeyEnter.String():
+				if len(m.getSelectedComponents()) > 0 {
+					var cmdsToRun []tea.Cmd
+
+					for _, listItem := range m.getSelectedComponents() {
+						// TODO: DRY up since we have this line copied from /internal/copier package
+						execCmd := exec.Command("uvx", "copier", "update", "-a", listItem.component.FileName, "-A")
+						cmdsToRun = append(cmdsToRun, updateComponent(execCmd))
+					}
+
+					cmd := tea.Batch(cmdsToRun...)
```

Comment:
This might work (totally guessing).

```suggestion
					cmd := tea.Sequence(cmdsToRun...)
```

Reply:
Ooh, nice - will try!

Reply:
Pretty sure that's working! Plus I got it to unselect components in list once process is done.